mavros/scripts/mavparam

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
```

Comment:
Sorry, it seems some local changes I did not wanted to include in the PR
were included. Should I make a new pull request?

On Fri, Aug 28, 2015 at 5:05 PM, Vladimir Ermakov <notifications@github.com>
wrote:

> In mavros/scripts/mavparam
> <https://github.com/mavlink/mavros/pull/377#discussion_r38238771>:
>
> > @@ -1,4 +1,4 @@
> > -#!/usr/bin/env python
> > +#!/usr/bin/env python2
>
> Changing default python is not case of that PR. Please revert that.
>
> Anyway scripts should work under py3 too, but that require to build ros
> with experimental py3 support.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mavlink/mavros/pull/377/files#r38238771>.
>